src/seedu/addressbook/parser/Parser.java

```diff
@@ -73,33 +74,36 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
+            case AddCommand.COMMAND_WORD:
```

Comment:
Are there tabs here (and in the following Case statements?)
Please see https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#layout